src/strawchemy/dto/types.py

```
@@ -181,40 +310,54 @@ class DTOConfig:
+    def _has_field_group(cls, value: FieldSpec | FieldIterable) -> bool:
+        """True if the selection contains a FieldGroup member."""
+        if isinstance(value, str):
+            return value in FieldGroup.values()
+        return any(isinstance(item, FieldGroup) for item in value)
 
     @classmethod
-    def _merge_field_iterables(cls, *iterables: IncludeFields | FieldIterable) -> IncludeFields | FieldIterable:
-        if any(iterable == "all" for iterable in iterables):
-            return "all"
-        return set().union(*iterables)
+    def _include_field(cls, field_name: str, is_relation: bool, fields: FieldSpec | FieldIterable | None) -> bool:
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)